raiden_installer/constants.py

```diff
@@ -0,0 +1,31 @@
+import sys
+from pathlib import Path
+from xdg import XDG_DATA_HOME, XDG_CONFIG_HOME
+
+
+'''
+Raiden installer uses the XDG Base Directory standard for
+storing its files. XDG_DATA_HOME is used for storing user
+keystore files whilst XDG_CONFIG_HOME stores config files
+necessary for initializing Raiden.
+'''
+BINARY_DIR = Path('/usr/local/bin')
+CONFIG_DIR = Path(XDG_CONFIG_HOME).joinpath('Raiden')
+CONFIG_DIR.mkdir(parents=True, exist_ok=True)
+KEYSTORE_DIR = Path(XDG_DATA_HOME).joinpath('Raiden')
+KEYSTORE_DIR.mkdir(parents=True, exist_ok=True)
+
+
+PLATFORM = 'macOS' if sys.platform == 'darwin' else 'linux'
+
+
+'''
```

Comment:
if you need to section your file using block quotes, consider splitting it up into submodules instead.